src/main/java/com/pingcap/tikv/types/BytesType.java

```diff
@@ -109,7 +109,7 @@ public static void writeBytes(CodecDataOutput cdo, byte[] data) {
    * @param cdo destination of data.
    * @param data is value that will be written into cdo.
    */
-  public static void writeCompactBytes(CodecDataOutput cdo, byte[] data) {
+  static void writeCompactBytes(CodecDataOutput cdo, byte[] data) {
```

Comment:
Please leave it public before refactory. Type class for now serve both as Type itself and encoder/decoder. So we use this method as utility.